### PR TITLE
[communication-sms] enable multiple sms broadcast

### PR DESCRIPTION
**Pull request by paolamvhz**

### CODE REVIEW

File: sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java [hunk 1]
```diff
@@ -39,56 +40,82 @@ public static void main(String[] args) {
         // Build a new SmsClient
         SmsClient smsClient = smsClientBuilder.buildClient();
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java:42]: Can we remove main and convert this to a createSmsClient method.. also the code around smsOptions is no longer needed in this

File: sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java [hunk 2]
```diff
@@ -39,56 +40,82 @@ public static void main(String[] args) {
...
         SendSmsOptions options = new SendSmsOptions();
         options.setEnableDeliveryReport(true);
 
-        // Send the message and check the response for a message id
-        SendSmsResponse response = smsClient.sendMessage(
-            new PhoneNumberIdentifier("<leased-phone-number>"), 
-            to, 
+    }
+
+    public void sendMessageToOneRecipient (SmsClient smsClient, SendSmsOptions options ){
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java:49]: Lets create the options here to show the usage.. set both enableDeliveryReport and tag

File: sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java [hunk 3]
```diff
@@ -39,56 +40,82 @@ public static void main(String[] args) {
...
+            "<leased-phone-number>",
+            to,
             "your message",
             options /* Optional */);
 
         System.out.println("MessageId: " + response.getMessageId());
+
     }
 
-    public void createSmsClienttWithConnectionString() {
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java:61]: nit: lets keep it at the top

File: sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java [hunk 4]
```diff
@@ -39,56 +40,82 @@ public static void main(String[] args) {
...
-        // Create an HttpClient builder of your choice and customize it
-        HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
+    public void sendMessageToMultipleRecipients (SmsClient smsClient, SendSmsOptions options){
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java:67]: Declare options within the method body

File: sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java [hunk 5]
```diff
@@ -39,56 +40,82 @@ public static void main(String[] args) {
...
-     * Sample code for creating a sync Communication Identity Client using AAD authentication.
-     *
-     * @return the Communication Identity Client.
-     */
-    public SmsClient createCommunicationIdentityClientWithAAD() {
-        // You can find your endpoint and access key from your resource in the Azure Portal
-        String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+    public void addATagForTheMessagesSent (SmsClient smsClient){
+        //Add a tag to your messages
+        List<String> toMultiplePhones = new ArrayList<String>();
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java:88]: We can remove this method, and add the tag in the sendSms samples

File: sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java [hunk 6]
```diff
@@ -39,56 +40,82 @@ public static void main(String[] args) {
...
+            options /* Optional */);
+
+        for (SendSmsResponseItem messageResponseItem
+            : responseMultiplePhones) {
+            System.out.println("MessageId sent to " + messageResponseItem.getTo() + ": " + messageResponseItem.getMessageId());
+        }
+    }
+
+
+    public void createCommunicationIdentityClientWithConnectionString() {
```
Comment (ankitarorabit) [inline sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java:109]: This should be SmsClient .. and move it at the top

**Comment by paolamvhz** [comment]
no longer need

**Pull request closed without merging**